wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -164,14 +126,21 @@ getAccounts = getAccountsIncludeUnready False
 
 getWalletIncludeUnready :: MonadWalletWebMode m => Bool -> CId Wal -> m CWallet
 getWalletIncludeUnready includeUnready cAddr = do
```

Comment:
Maybe create a new datatype for `includeUnready` and replace `Bool` in order to make it more (type)safe and understandable?

```
data WalletInclusionState = WalletReady | WalletNotReady
```

Reply:
I agree, but this code was written many months ago and it's out of scope of this PR.
Although would be nice to change it in some way when someone has extra time